platform/security/security-oidc-realm/src/main/java/org/codice/ddf/security/oidc/realm/OidcCredentialsResolver.java

```diff
@@ -137,14 +165,80 @@ public void validate(OidcCredentials credentials, WebContext webContext) {
     }
   }
 
+  private void createClientAuthentication() {
```

Comment:
📘 Copied from `OidcAuthenticator` code. More robust way to put together the client authentication information from the `OidcMetadata` (for example: client id and secret)